packages/react/src/components/AccountSettings/DeleteUser/defaultComponents.tsx

```diff
@@ -0,0 +1,55 @@
+import { translate } from '@aws-amplify/ui';
+import React from 'react';
+
+import { Card, Flex, Text, Button, Alert } from '../../../primitives';
+import { AccountSettingsError, AccountSettingsSubmitButton } from '../types';
+import { DeleteUserWarningProps } from './types';
+
+export const DefaultSubmitButton: AccountSettingsSubmitButton = ({
```

Comment:
Just curious, is this component overridable?

Reply:
It will be -- but on a later PR.

Reply:
Can be done in that PR, but unless we have specific functionality or styling we probably don't need a standalone component or can just assign this as:
```
const DefaultSubmitButton: AccountSettingsButton = Button
```